include/haproxy/global-t.h

```diff
@@ -194,7 +194,8 @@ struct global {
 			int disk_loader;                 /* the number of files load once */
 			int disk_saver;                  /* the number of entries checked once for persist_async */
 			int clean_temp;                  /* clean temp file or not */
-
+			int disk_always_read;
```

Comment:
It's a little bit confusing, as this one only affects whether or not to check the existence of the cache file on disk if disk-load is done. Do you have other candidates for the var?

Reply:
What do you think of `check_disk_after_load`? a little bit long though

Reply:
How about `alway_check_disk`?

Reply:
`always` ? Good idea. there's no need to emphasize `after loaded` indeed. Let's go with `always_check_disk`.